kafka-lib_0_10/src/main/scala/Consumer.scala

```diff
@@ -6,6 +6,16 @@ import scala.util.Try
 trait Consumer[T] {
 
   def kafkaConsumer: KafkaConsumer[String, String]
+  
+  def processBatch(
```

Comment:
Hmmm, after looking at the other PR, I understand this makes it easier to use. We can either add default implementations which just throws `UnsupportedOperationException`, or leave it for now but change the major version to reflect that it's a breaking change.